contrib/systemd/ocid.service

```diff
@@ -8,7 +8,7 @@ Type=notify
 EnvironmentFile=-/etc/sysconfig/ocid-storage
 EnvironmentFile=-/etc/sysconfig/ocid-network
 Environment=GOTRACEBACK=crash
-ExecStart=/usr/bin/ocid \
+ExecStart=/usr/local/bin/ocid \
```

Comment:
right, but we provide `make install.systemd` target as well and running `systemctl start ocid` fails w/o this patch and it seems weird